src/routes/comments/index.ts

```diff
@@ -119,14 +137,14 @@ export function createPropertyComment() {
         compound: mood.compound,
       },
       createdDate: new Date().toISOString(),
-    };
+    });
```

Comment:
Let's not save mood.type to database, since it is just a display representation
we can map the ML data to moodType in toResponse

For example, with the bug that was fixed in PR, the mood.type is incorrect for many entries in the database. If we map them in toResponse, this would not matter